Nowra SOP Updates (#771)
vatpac-technology/sops: docs/enroute/melbourne/WOL.md

### YSNW
WOL is responsibile for facilitating operations in and out of YSNW.

## Local Procedures
### Special Use Airspace

There are multiple volumes of [SUA](../../../controller-skills/sua) within WOL airspace associated with military operations in and out of YSNW.

<figure markdown>
![Notable SUA in WOL Airspace](../img/wol_sua.png){ width="700" }
  <figcaption>Notable SUA in WOL Airspace</figcaption>
</figure>

Each TCU must [give heads up coordination](../../../controller-skills/coordination/#airways-clearance) with the relevant enroute controllers **prior** to any departures intending to operate in a currently inactive SUA.

!!! phraseology

>     <span class="hotline">**NWA** -> **WOL**</span>: "On the groud YPTN, SWFH12, requests activation of R420F `F125-F300`, from 0300 until 0500.”  

Reviewer: ```suggestion
    <span class="hotline">**NWA** -> **WOL**</span>: "On the groud YSNW, SWFH12, requests activation of R420F `F125-F300`, from 0300 until 0500.”  
```

    <span class="hotline">**WOL** -> **NWA**</span>: "SWFH12, expect activation of R420F `F125-F300` at 0300 until 0500."   
    <span class="hotline">**NWA** -> **WOL**</span>: "SWFH12."   

Non-participating aircraft intending to transit an activated SUA should be rerouted, where possible, [subject to the VATSIM Code of Conduct](../../../sua/#ad-hoc-activations).

#### R420F Nowra

> The R420F Nowra [restricted area](../../../controller-skills/sua/#restricted-areas) are located over YSNW, `F125-F300`, located entirely in WOL airspace. 

Reviewer: ```suggestion
The R420F Nowra [restricted area](../../../controller-skills/sua/#restricted-areas) is located over YSNW, `F125-F300`, located entirely in WOL airspace. 
```

The resricted area sits directly above the NWA TMA and when NWA is online aircraft will be transferred directly to/from the SUA. When [NWA is offline](#reclassifications), aircraft will contact WOL for transit through the surrounding civilian airspace.

##### Affected Civil Operations

> When activated, the restricted areas disrupt traffic on the **H20** and **H536** high altitude airways which are used by aircraft travelling between YSSY and Tasmania.

Reviewer: ```suggestion
When activated, the restricted area disrupts traffic on a number of high and low altitude airways, including **H20**, **H536**, **V501**, **W144**, **W436**, and **W575**, used primarily by aircraft departing the Sydney Basin.
```

Aircraft may be given an additional requirement to climb above the vertical limits of the activation, or rerouted manually to avoid the area.

!!! tip  
    Activation of the R420F **significantly increases** the workload for **WOL** and **SDS**, as the airspace available for YSSY departures via WOL. During peak periods (such as [Milk Run Monday](../../../events/milkrun)), activation of the restricted area should not be approved.
    
#### M440-P Tasman Sea
The M440-P Tasman Sea [MOAs](../../../controller-skills/sua/#military-operating-areas) are located east of YSNW and extend into the Tasman Sea, `NOTAM-NOTAM`, located entirely in WOL airspace. 

> The restricted area sits directly above the NWA TMA and when NWA is online aircraft will be transferred directly to/from the SUA. When [NWA is offline](#reclassifications), aircraft will contact WOL for transit through the surrounding civilian airspace.

Reviewer: ```suggestion
#### M440A-P Tasman Sea
The M440A-P Tasman Sea [MOAs](../../../controller-skills/sua/#military-operating-areas) are located east of YSNW and extend into the Tasman Sea, `NOTAM-NOTAM`, located entirely in WOL airspace. 

The restricted areas sit directly above, and adjacent to, the NWA TMA and when NWA is online aircraft will be transferred directly to/from the SUA. When [NWA is offline](#reclassifications), aircraft will contact WOL for transit through the surrounding civilian airspace.
```
